test/integration/apiserver/apply/apply_crd_test.go

```diff
@@ -191,12 +191,15 @@ func TestApplyCRDStructuralSchema(t *testing.T) {
 										"type": "string"
 									},
 									"protocol": {
-										"type": "string",
-										"nullable": true
+										"type": "string"
 									}
 								},
 								"required": [
-									"containerPort"
+									"containerPort",
```

Comment:
Basically, the integration test fails if they're not required, or don't have a default due to the new changes.

Reply:
the validation is not enforcing required-or-default on the right properties... see  https://github.com/kubernetes/kubernetes/pull/88076/files#r385693745

once that is fixed I expect you can revert most of the changes in this test

Reply:
only list-map-key properties (containerPort and protocol) should have needed to be required-or-defaulted... if you had to change anything else that sounds like a bug in the validation

Reply:
Okay, `protocol` is not required and it has no defaults. The test uses the `v1beta1` group which doesn't allow defaults so that means `protocol` has to be required.